javascript/javascript2/week1/shortestword.js

```diff
@@ -0,0 +1,18 @@
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function checkTheShortestWord(){
+    let shortestWord;
+    let shortestWordLength=danishWords[0].length;
+
+    for(let i=0;i<danishWords.length;i++){
```

Comment:
In the first loop iteration your code will always compare the first word with itself!
E.g., here in the first iteration both word and shortestWord are "bil"
Which means there is always one unnecessary iteration. Can you change the for loop such that this is solved?